libs/s25main/desktops/dskCredits.h

```diff
@@ -57,10 +64,10 @@ class dskCredits : public Desktop
         std::string lastLine;
         glArchivItem_Bitmap* pic;
         std::vector<Line> lines;
-        explicit CreditsEntry(std::string title, std::string lastLine = "")
+        explicit CreditsEntry(std::string title, std::string lastLine = std::string())
```

Comment:
Why this change? This is just more noise. Use `{}` if you don't like an empty string

Reply:
you shall not use `""` - it will/can create a copy from the constant `""`. but `{}` is propably the  best idea. will change that

Reply:
So what is the difference? And don't see any problem.

Reply:
At any optimization level (hence above `-O0`) the code is literally the same: https://godbolt.org/z/7dGAza

But readability is different:
- `void foo(std::string s = "")` - "Default value is an empty string"
- `void foo(std::string s = std::string())` - "Default value is string...with no value"
- `void foo(std::string s = {})` - "Default value is an default constructed...string"

So the first is easiest to parse visually. the second needs you to go through a lot of chars to see what the actual value is and the extra parens don't help and the last needs you to go back to see what thing you are creating and then see it's an empty string.